dev-tools/packaging/package_test.go

```diff
@@ -301,11 +306,15 @@ func checkNpcapNotices(pkg, file string, contents io.Reader) error {
 	return nil
 }
 
-func checkDocker(t *testing.T, file string) {
+func checkDocker(t *testing.T, file string) (string, int64) {
+	if !strings.HasPrefix(file, "edot-collector") {
+		return checkEdotCollectorDocker(t, file)
+	}
```

Comment:
aside from the "special checks" that an Edot collector docker image has here, shouldn't the condition be 
```suggestion
	if strings.HasPrefix(file, "edot-collector") {
		return checkEdotCollectorDocker(t, file)
	}
```